src/main/java/seedu/address/model/delivery/Delivery.java

```diff
@@ -15,7 +18,7 @@
 public class Delivery {
 
     private final DeliveryId deliveryId;
-    private final ItemName itemName;
+    private final Set<ItemName> items = new HashSet<>();
```

Comment:
Is there a reason why `HashSet` was chosen over a structure such as `List`? 🤔

Reply:
The way I think about it is why would you add the same item multiple times? Hence why it's a HashSet currently. But do let me know how you feel about this. Should it remain as it is or do you have another opinion on this?

Reply:
Thank you for clarifying! I understand why you would want to use a `HashSet` now 👍
Perhaps we can also implement a way to track the count of each item within the `HashSet`.